src/Components/People.js

```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import React, { useState } from "react";
+const People =()=>{
+    const [input,setInput] = useState('')
+    const [people,setPeople] = useState({})
+    const [hasSearch,setHasSearch] = useState(false)
+    const handleSubmit = async (e) => {
+        e.preventDefault();
+        setHasSearch(true)
+        try {
+            const res = await axios.get(
+                `https://ghibliapi.herokuapp.com/people/?name=${input}`
+            )
+            setPeople(res.data);
```

Comment:
What's the data type of res.data? 
Should it be an array? 
If you're planning to keep it as an object in state (which I like), why not grab the element out of the array here? 
res.data[0]